src/index.json

```diff
@@ -288,6 +288,38 @@
                     "version": "0.1.1"
                 },
                 "sha256Digest": "6a03b2b1a78f44e096650870d02fc2d86ee30fb07a42656716f1ced9d8651831"
+            },
+            {
+                "downloadUrl": "https://azurecliprod.blob.core.windows.net/cli-extensions/application_insights-0.1.2-py2.py3-none-any.whl",
+                "filename": "application_insights-0.1.2-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "azext.minCliCoreVersion": "2.0.56",
```

Comment:
The dependency should be 2.0.78 as this is the version of the CLI that has the app-insights endpoint.